docs/onboarding/1_environment_setup.md

```diff
@@ -0,0 +1,48 @@
+# Environment setup
+This document may be incomplete now or in the future, so if you faced any problems ask the team for help.
+
+### Mac
+1. Install a brew, it's not mandatory but may be convenient for installing other software.
+1. Currently, the zsh is default shell on a mac. If you prefer bash use `chsh -s /bin/bash`.  
```

Comment:
`is the default`

Reply:
nope

Reply:
![image](https://user-images.githubusercontent.com/1173057/88285528-419e4c80-ccbd-11ea-9df4-9c9880a04c84.png)

I think pawel is correct. Grammerly.com is helpful for issues like this.

I'd write it like this:
> Zsh is the default shell on macOS.

Reply:
Right I misunderstood the comment. I will fix it in another PR.